app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from typing import Any, Callable
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_box = {}
+
+    def wrapper(*args: Any, **kwargs: Any) -> Any:
+        key = (args, frozenset(kwargs.items()))
+        if key in cache_box:
+            print("Getting from cache")
```

Comment:
Code Cleanliness issue: It is good practice to remove debug prints from your code before committing. The print statement on this line might have been useful for debugging, but it can be removed now.